account-keys/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "mc-account-keys"
-version = "1.3.0-pre0"
+version = "2.0.0"
```

Comment:
IIUC these should be `3.0.0-pre0`

Reply:
I don't think so, we should do that when we start releasing 3.0.0, at least that's how I understand the process right now